tautaunn/tasks/datacards.py

```diff
@@ -504,25 +632,26 @@ def card_names(self):
 
         return self._card_names
 
-    def requires(self):
-        return {
-            skim_name: EvaluateSkims.req(self, skim_name=skim_name)
-            for skim_name in self.skim_names
-        }
 
     def store_parts(self):
         parts = super().store_parts()
         parts.insert_before("version", "ensemble", self.get_model_name())
         return parts
 
+
     def output(self):
         # prepare the output directory
         dirname = f"{self.binning}{self.n_bins}"
-        if self.binning in ["ud", "ud_flats", "tt_dy_driven"]:
-            dirname += f"_{self.uncertainty}_{self.signal_uncertainty}"
         if self.output_suffix not in ("", law.NO_STR):
             dirname += f"_{self.output_suffix.lstrip('_')}"
         d = self.local_target(dirname, dir=True)
+        # hotfix location in case TN_STORE_DIR is set to Marcel's
```

Comment:
@JohanWulff We removed this hotfix for now but feel free to change the "DEFAULT_STORE" attribute of the task to your value.